src/main.py

```diff
@@ -1,17 +1,44 @@
-from typing import Optional
-
-
-def print_something(something: Optional[str]) -> None:
-    print(something)
-
-
-def main():
-    print_something("Welcome to ingeSTation project!")
-
-
-def ingestation():
-    return main()
+# from modules.configs import MergedConf
```

Comment:
We need a very good reason to merge commented code to main branch. We will not merge commented code to any stable branch.